scrapy/czlscrape/spiders/cultura.py

```diff
@@ -24,16 +24,24 @@
 $                                           # end of string
 '''
 
-# This regex is not expressed in verbose mode because it simply doesn't work
-# that way
-FEEDBACK_DEADLINE_DATE_PATTERN = r'.*(?:pana la data de|pana la data) (?P<day>\d{1,2})\.(?P<month>\d{1,2})\.(?P<year>\d{4}).*'
+FEEDBACK_DEADLINE_DATE_PATTERN = r'''
```

Comment:
Schimbarea asta trebuie insotita de o schimbare in codul care o foloseste, specificand ca e un regex `VERBOSE`.